src/main/java/com/example/yatzysimulator/controller/YatzyController.java

```diff
@@ -15,15 +15,20 @@ public class YatzyController {
     @Autowired
     private YatzyService service;
 
-    @GetMapping("roll")
-    public List<Integer> roll(){
-        return service.rollDice();
+    @GetMapping("roll/{token}")
+    public @ResponseBody DiceResponse roll(@PathVariable String token){
+        return service.rollDice(token);
     }
     @PostMapping("roll")
```

Comment:
this method is mostly a calculation method, so the path should be `calculation`  not `score`